src/core/list/components/Title.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Box, Typography } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+
+const primary = {
+  fontSize: 36,
+  fontWeight: '700',
+  lineHeight: 1.11,
+  letterSpacing: '-0.03em',
+  textAlign: 'left',
```

Comment:
Isn't left the default ? I don't think you need to specifically mention it unless you overwrite something

Reply:
Done.